clion/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ intellij {
 
 dependencies {
     implementation(project(":common"))
-    implementation("org.sonarsource.sonarlint.core:sonarlint-core:$sonarlintCoreVersion")
+    implementation("org.sonarsource.sonarlint.core:sonarlint-embedded:$sonarlintCoreVersion")
```

Comment:
CFamilyLanguageActivator is overriding void amendLanguages(Set<org.sonarsource.sonarlint.core.commons.Language> enabledLanguages, boolean isConnected) of LanguageActivator which exists in the common package. So I would avoid making this change now since it seems it requires more work than needed on Eclipse